4_Methodology/BS_SD.adoc

```diff
@@ -757,46 +766,48 @@ The evaluator shall report the summary of result of EA defined above, especially
 
 The same <<Toolbox>> can also be used for AVA_VAN.1 evaluation (i.e. penetration testing) because AVA_VAN.1 requires the evaluator to devise tests based on information available in the public domain. However, <<Toolbox>> should be used in a different manner for AVA_VAN.1 evaluation. The following section explains how <<Toolbox>> should be used in EAs for ATE_IND.1 and AVA_VAN.1.
 
+<<Toolbox>> is intended to be a state of the art set of test and is continually updated. The latest versions of the <<Toolbox>> for each supported modality must be used by the evaluator at the time the testing begins.
+
 ==== Presentation Attack Instrument (artefact) species
 There are many types of Presentation Attack Instruments that can be used to test a PAD system. The <<BIOPP-Module>> specifically defines the artefacts that are to be used as artificial, and not natural. Natural artefacts, such as a dead eye, are not considered in scope for this evaluation. When searching for new artefact species, only artificial species should be considered.
 
 === EAs for ATE_IND.1 (Independent testing - conformance)
 
-==== Independent test activities using <<Toolbox>>
+==== Independent test activities using Toolbox
 
 As described in previous section, <<Toolbox>> defines test items to create a representative set of artefacts that the evaluator shall use for the testing. During ATE_IND.1 evaluation, the evaluator shall conduct all test items in <<Toolbox>> for the selected modalities without any change. The evaluator is not allowed to skip any test items in the <<Toolbox>> to maintain compatibility between different evaluations.
 
 During the independent testing, the evaluator may find artefacts that are incorrectly matched to the enrolled target user however, the evaluator may not be able to reliably reproduce a successful presentation attack.
 
-<<Toolbox>> defines the Pass/Fail criteria, maximum imposter attack presentation match rate for artefacts. The evaluator shall follow the <<Toolbox>> criteria for the number of artefact presentations and confirm that the TOE’s match rate is below the specified criteria during the independent testing. The evaluator shall assign a fail verdict to those TOE that doesn’t satisfy the criteria.
+<<Toolbox>> defines the Pass/Fail criteria, maximum imposter attack presentation match rate for artefacts. The evaluator shall follow the <<Toolbox>> criteria for the number of artefact presentations and confirm that the TOE’s match rate is below the specified criteria during the independent testing. The evaluator shall assign a fail verdict to those TOE that don’t satisfy the criteria.
 
 The artefacts that pass the criteria but show the higher imposter attack presentation match rate will be tested again during the AVA_VAN.1 evaluation.
```

Comment:
Is it correct that this is the case? If new artefacts are found it doesn't seem to require using the existing toolbox tests to check, even if there aren't many new (i.e. say only one new artefact is found).

If this really is a "will be tested" then we should adjust the section about what to test to include the highest likelihood artefacts here as well as anything new.

Reply:
Is this a comment for line 801? I am not sure that my comment below correctly deal with your concern but see below.

I think that we should update the toolbox as follows.

1. If the evaluator can find a new artefact that are not included in the toolbox during the penetration testing, the evaluator shall inform the BIO-iTC (See 2nd para in 6.3.1.3.2. New artefacts found test plan [BIOSD])
2. the BIO-iTC verify that the new artefact is actually within the Basic attack potential and add it to the toolbox 
3. the BIO-iTC informs the toolbox update to CCRA

Reply:
Sorry, the comment didn't link very well. Yes, this is about line 801. If you also look at the section starting at 

https://github.com/biometricITC/cPP-biometrics/blame/eeb8a1ff3903f88a4eb55e59f6c9bab1d81810ca/4_Methodology/BS_SD.adoc#L911

the test plan for new artefacts found implies that you then ONLY test the new artefacts. There isn't anything that states that if you find there to be new artefacts that you should test those in addition to the most likely candidates from the IND testing, which I think is what we actually want. 

So what I'm really getting to is that if you can find new artefacts to test, and we only look at those, then saying that this potential candidate "will be tested" is incorrect since it wouldn't be covered under the test plan. 

The way we have the test plans laid out, there is one set of steps to follow if you have new artefacts on one if you don't. I think that what we really want is one test plan that ALWAYS covers these likely candidates, and IF new artefacts are found, also tests those. Today though, I read it as two independent test plans/paths.

So if we MUST always test these high potential candidates from the IND, then we need to adjust how the AVA_VAN test plan is written out so ensure both cases are tested in all scenarios. Or we need to say that these high potential candidates MAY be tested.

Reply:
@woodbe 
I created PR #269 to update your editorial-updates branch for the above issue. Please take a look at it and tell me if it can't work.